tests/test_transforms/test_timestamp/test_holiday_transform.py

```diff
@@ -201,11 +285,21 @@ def test_holidays_min(iso_code: str, answer: np.array, two_segments_simple_ts_mi
         (pd.date_range(start="2019-11-25", end="2021-02-25", freq="M")),
     ),
 )
-def test_holidays_failed(index: pd.DatetimeIndex, two_segments_simple_ts_daily: TSDataset):
-    ts = two_segments_simple_ts_daily
-    ts.df.index = index
+def test_holidays_failed(index: pd.DatetimeIndex, two_segments_simple_ts_day_15min: TSDataset):
```

Comment:
We should rewrite this test, because now it checks only `two_segments_simple_ts_day_15min`. Try to rewrite it like there: https://github.com/etna-team/etna/blob/aa9890a79fe9f2b9922073f961de1edc1951e0b6/tests/test_pipeline/test_pipeline.py#L508
We should use dataset as parameter.